### PR TITLE
import quest items

### PR DESCRIPTION
**Pull request by Razzmatazzz**
Initial PR for importing quest items but excluding them from scanner batches and regular items in the API.

### CODE REVIEW

**Comment by Razzmatazzz** [comment]
.deploy

**Comment by Razzmatazzz** [comment]
deploy

**Comment by Razzmatazzz** [comment]
.deploy

**Review by GrantBirki — DISMISSED** [review]

File: src/tarkov-data-manager/jobs/update-item-cache.js [hunk 1]
```diff
@@ -292,7 +292,7 @@ module.exports = async () => {
             disabledItemIds: [...itemMap.values()].filter(item => item.types.includes('disabled')).map(item => item.id)
         });
         for (const [key, value] of itemMap.entries()) {
-            if (value.types.includes('disabled')) continue;
+            if (value.types.includes('disabled') || value.types.includes('quest')) continue;
             if (!bsgItems[key] && !presets[key]) continue;
```
Comment (Shebuka) [inline src/tarkov-data-manager/jobs/update-item-cache.js:296]:
```suggestion
            if (value.types.includes('disabled') || value.types.includes('quest'))
                continue;
            if (!bsgItems[key] && !presets[key])
                continue;

```

**Review by Shebuka — APPROVED** [review]

**Review by Shebuka — APPROVED** [review]

**Pull request merged**